indra/newview/llexternaleditor.cpp

```diff
@@ -44,8 +44,20 @@ LLExternalEditor::EErrorCode LLExternalEditor::setCommand(const std::string& env
     std::string cmd = findCommand(env_var, override);
     if (cmd.empty())
     {
-        LL_WARNS() << "Editor command is empty or not set" << LL_ENDL;
-        return EC_NOT_SPECIFIED;
+        LL_INFOS() << "Editor command is empty or not set, falling back to OS open handler" << LL_ENDL;
+#if LL_WINDOWS
+        static const std::string os_cmd = "C:\\Windows\\explorer.exe \"%s\"";
```

Comment:
Probably better to use
```suggestion
        static const std::string os_cmd = "%SystemRoot%\\explorer.exe \"%s\"";
```
It is not guaranteed Windows is always installed at C:\Windows